front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -584,10 +567,9 @@ const AdminProjectsProjectGeneral = () => {
           />
         )}
 
-        <HeaderImageDropzone
-          projectHeaderImage={projectHeaderImage}
-          handleHeaderOnAdd={handleHeaderOnAdd}
-          handleHeaderOnRemove={handleHeaderOnRemove}
+        <HeaderBgInput
+          imageUrl={project?.attributes.header_bg.large}
```

Comment:
We can narrow down the type for the imageUrl to just `string` or `null`. (Not much of difference here, but generally it's good to narrow down types as much as possible.)